src/pages/project.js

```diff
@@ -9,10 +9,25 @@ import Breadcrumbs from "components/Breadcrumbs";
 import ProjectBreadcrumb from "components/Breadcrumbs/Project";
 import ProjectDetailsSidebar from "components/ProjectDetailsSidebar";
 import Environments from "components/Environments";
+import ProjectNavTabs from "components/ProjectNavTabs";
 import withQueryLoading from "lib/withQueryLoading";
 import withQueryError from "lib/withQueryError";
 import { withProjectRequired } from "lib/withDataRequired";
-import { ProjectDetailsWrapper } from "../styles/pageStyles";
+import { ProjectDetailsWrapper, ProjectWrapper } from "../styles/pageStyles";
+
+const getProductionEnvironments = (environments) => {
+  if (!environments) return null;
+  return environments.some((e) => e.environmentType === "production")
+    ? environments.filter((e) => e.environmentType === "production")
+    : false;
+};
+
+const getDevelopmentEnvironments = (environments) => {
```

Comment:
Instead of having 2 different functions called `getProductionEnvironments` and `getDevelopmentEnvironments `, I think it would make sense to have one for both, and just use one array method like so:

```suggestion
const getEnvironmentsByType = (environments, type) => {
    if (!environments) return null;
    const filtered = environments.filter((e) => e.environmentType === type);
    
    return filtered.length ? filtered : false
    
}
```

Reply:
Removed these functions entirely, they're legacy functions from the PR I built upon for the Project overview changes. I just missed removing them!